libs/core/src/lib/api/billing.service.ts

```diff
@@ -95,17 +113,27 @@ export class BillingService {
         ),
       ),
       map((data) => {
+        let subscription: AccountSubscription;
         if (!data.hasOwnProperty('provider')) {
           // Backward compatibility: when there is no provider, it's an old STRIPE subscription
-          return {
+          subscription = {
             provider: 'STRIPE',
             subscription: data,
           } as AccountSubscription;
         } else {
-          return data as AccountSubscription;
+          subscription = data as AccountSubscription;
         }
+
+        this._accountSubscriptionInit = true;
+        this._accountSubscriptionCache.next(subscription);
+        return subscription;
+      }),
+      catchError(() => {
+        if (fromCache) {
+          this._accountSubscriptionInit = true;
```

Comment:
so if we get an error on the first call, we will return `null` forever?

Reply:
yes